### ISSUE TITLE
Hyperparameter search

### ISSUE THREAD

**Issue by sepmein** [sepmein/turtle#4]
train model multiple times to search best hyper parameter to fit the cross validation set.
